.github/workflows/release.yaml

```diff
@@ -124,7 +124,7 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: |
-            us-docker.pkg.dev/edgebit-containers/containers/odyn
```

Comment:
If these are for pushing (I think so?) they need to go through the normal hostname. `registry.edgebit.io` is for consumption only.

Reply:
[Resolved](https://github.com/edgebitio/enclaver/compare/fb6c72fb593ce5f287c57b677cd037ee53551ebb..77b35ba9c807c7743242fef1fc942763361908b4)